packages/jest-cli/src/Runtime/Runtime.js

```diff
@@ -23,7 +37,34 @@ const normalizedIDCache = Object.create(null);
 const unmockRegExpCache = new WeakMap();
 
 class Runtime {
-  constructor(config, environment, resolver) {
+  _config: Config;
+  _CoverageCollector: any;
+  _coverageCollectors: Object;
+  _currentlyExecutingModulePath: string;
+  _environment: Environment;
+  _explicitShouldMock: Object;
+  _isCurrentlyExecutingManualMock: ?string;
+  _mockFactories: Object;
+  _mockMetaDataCache: Object;
+  _mockRegistry: Object;
+  _mocksPattern: ?RegExp;
+  _moduleRegistry: Object;
+  _resolver: Object;
+  _shouldAutoMock: boolean;
+  _shouldMockModuleCache: Object;
+  _shouldUnmockTransitiveDependenciesCache: Object;
+  _testRegex: RegExp;
+  _transitiveShouldMock: Object;
+  _unmockList: ?RegExp;
+  _virtualMocks: Object;
```

Comment:
We need to make all of these types a bit more specific but we can do that in a follow-up.